addons/html_editor/static/src/others/dynamic_field/dynamic_field_plugin.js

```diff
@@ -0,0 +1,341 @@
+import { isHtmlContentSupported } from "@html_editor/core/selection_plugin";
```

Comment:
merge this plugin with dynamic table_plugin
they both handle add field in their specific details